js/modules/k6/http/http.go

```diff
@@ -67,10 +105,14 @@ type HTTP struct {
 	OCSP_REASON_REMOVE_FROM_CRL        string `js:"OCSP_REASON_REMOVE_FROM_CRL"`
 	OCSP_REASON_PRIVILEGE_WITHDRAWN    string `js:"OCSP_REASON_PRIVILEGE_WITHDRAWN"`
 	OCSP_REASON_AA_COMPROMISE          string `js:"OCSP_REASON_AA_COMPROMISE"`
+
+	responseCallback func(int) bool
 }
 
+// New ...
+// TODO deprecate this method
```

Comment:
Is there a reason for not using this in `NewModuleInstance()`? Or for not deleting it now, if you don't want to use it?

Reply:
predominantly going through all the code and changing ... in case we decided that this was not going to be the solution ... also initially I thought there are like 50 places where it is used, but it's a lot less actually ... So I am probably going to drop it.